fat/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.12.1
 
+- Integrate upstream `rust-libp2p` `0.54` changes to the bootstrap process
 - Add back origin attribute to metrics
 - Update avail-light-core to 1.0.3
```

Comment:
Since we updated version of the avail-light-core, thats should be reflected in changelog. Previous update is not relevant anymore. What is changed in avail-light-core can be explored from the root CHANGELOG.md when relevant revision of the released fat client is browsed.
 
```markdown
- Update avail-light-core to 1.0.4
- Add back origin attribute to metrics
```